config-templates/ibm/ibm-csi-block/1.4.0/README.md

```diff
@@ -0,0 +1,101 @@
+# IBM block storage Container Storage Interface driver
+
+IBM block storage CSI driver is based on an open-source IBM project, included as a part of IBM Storage orchestration for containers. IBM Storage orchestration for containers enables enterprises to implement a modern container-driven hybrid multicloud environment that can reduce IT costs and enhance business agility, while continuing to derive value from existing systems.
+
+For full release notes, compatiblity, installation, and user information, see the IBM block storage CSI driver documentation: https://www.ibm.com/support/knowledgecenter/SSRQ8T_1.4.0/
+
+## Prerequisites
+
+For full prerequisite instructions, see the following documentation, referred to in the introductory paragraph. 
+**Installation** > **Compatibility and requirements**.
+
+**Important:** Be sure to complete all prerequisite and installation steps before assigning hosts to your location. Do not create a Kubernetes cluster. This is done through Satellite.
+
+## IBM block storage CSI driver parameters & how to retrieve them
+
+
+
+Retrieve all parameters required by this template.
+
+```
+
+ibmcloud sat storage template get --name ibm-csi-block --version 1.4.0
+
+```
+
+ **IBM block storage CSI driver parameters**
+| Parameter | Required? | Description | Default value if not provided |
+| --- | --- | --- | --- |
+| `namespace` | Optional | Deployment namespace. | `default` |
+| `sc-name` | Required | Storage class name to be created | N/A |
+| `space-efficiency` | Optional | Space efficiency of volume to be created | N/A |
+| `pool` | Required | Pool ID of volume to be created in | N/A |
+| `secret-name` | Required | Existing secret name | N/A |
+| `secret-namespace` | Required | Existing secret name namespace | N/A |
+| `fstype` | Optional | File system type | `ext4` |
+| `prefix` | Optional | Prefix name of volume to be created | N/A |
+| `VolumeExpansion` | Optional | Is volume expansion allowed? | `false` |

```

Comment:
```suggestion
| Parameter | Required? | Description | Default value if not provided |
| --- | --- | --- | --- |
| `namespace` | Optional | The namespace where you want to create the deployment. | `default` |
| `sc-name` | Required | The name of the storage class name that is created. | N/A |
| `space-efficiency` | Optional | The space efficiency of the volume that is created. Example: | N/A |
| `pool` | Required | The ID of the worker pool where you want to create the volume. Example:  | N/A |
| `secret-name` | Required | The name of your existing Kubernetes secret. | N/A |
| `secret-namespace` | Required | The namespace that your Kubernetes secret is in. | N/A |
| `fstype` | Optional | The file system type. Specify...  | `ext4` |
| `prefix` | Optional | The prefix name of the volume that is created. Example: | N/A |
| `VolumeExpansion` | Optional | Specify `true` to allow volume expansion or `false` to disallow volume expansion. | `false` |

```

Reply:
- Do the secret & deployment namespace need to be the same?
- What are some examples of the `space-effeciency` parameter? Or a link to review examples.
- Is pool the worker pool id?
- What are possible fstypes
- What are possible prefixes or examples

Reply:
done
instead of adding examples (that eventually mislead due to other options that will not be included in them) I added a link for this step in our documentation that include explanation on all fields.